internal/app/dogma/util.go

```diff
@@ -277,8 +276,16 @@ func marshalIndentObject(data interface{}) ([]byte, error) {
 	return json.MarshalIndent(data, "", "  ")
 }
 
-func marshalIndent(src []byte) ([]byte, error) {
-	dst := new(bytes.Buffer)
-	err := json.Indent(dst, src, "", "  ")
-	return dst.Bytes(), err
+func safeMarshalIndent(src []byte) ([]byte) {
+	if isJSON(src) {
+		dst := new(bytes.Buffer)
+		_ = json.Indent(dst, src, "", "  ")
+		return dst.Bytes()
+	}
+	return src
+}
+
+func isJSON(b []byte) bool {
+	var holder map[string]interface{}
+	return json.Unmarshal(b, &holder) == nil
```

Comment:
I think in this case we can use `json.Valid` to validate `b`. (https://golang.org/pkg/encoding/json/#Valid)

Reply:
Ah let me use that. Thanks!